docs/source/commandline/commandline-all.rst

```diff
@@ -976,6 +1724,10 @@ Core State/Playback Options
 
     The default is OFF (**-noburnin**).
 
+    Example:
+        .. code-block:: bash
+
+            mame64 neogeo -burnin
```

Comment:
This would usually be used with `-nothrottle` and `-str` to automatically get the burning image from a few attract loops.